tests/hash-backend-performance/rollup.config.ts

```diff
@@ -8,7 +8,7 @@ const OUT_DIR = "dist/esm";
 
 const bundles: RollupOptions[] = [
   {
-    input: "src/main.ts",
+    input: ["src/main.ts"],
```

Comment:
Accidentally committed this, but it doesn't hurt, either, as it's easier to extend 😄